docs/user-guide/api-mediation/authenticating-with-personal-access-token.md

```diff
@@ -22,8 +22,9 @@ APIs for security administrators are protected by SAF resource checking and can
    * [Invalidate all tokens for a service](#invalidate-all-tokens-for-a-service)
    * [Evict non-relevant tokens and rules](#evict-non-relevant-tokens-and-rules)
 
-:::note
-An SMF record can be issued when a Personal Access Token is generated. For more information, see [SMF records issued by API ML](api-mediation-smf.md)
+:::note Notes
+* An SMF record can be issued when a Personal Access Token is generated. For more information, see [SMF records issued by API ML](api-mediation-smf.md).
+* To enable Personal Access Token support when using the Caching Service, **Infinispan** is the required storage solution. Infinispan is part of Zowe installation. No additional software or installation is required when using this storage solution.
```

Comment:
sorry to backtrack but I think this should belong in a prerequisites section? Or required configuration? rather than a note